### PR TITLE
Handle gamma-control-v1 set_gamma events

### PR DESCRIPTION
**Pull request by emersion**
References: https://gitlab.freedesktop.org/wlroots/wlroots/-/merge_requests/4046
Depends on: #7493

### CODE REVIEW

**Comment by Ferdi265** [comment]
Fixes #7482

**Comment by Nefsen402** [comment]
LGTM

File: sway/desktop/output.c [hunk 1]
```diff
@@ -1076,6 +1091,16 @@ void handle_output_layout_change(struct wl_listener *listener,
 	update_output_manager_config(server);
 }
 
+void handle_gamma_control_set_gamma(struct wl_listener *listener, void *data) {
+	struct sway_server *server =
+		wl_container_of(listener, server, gamma_control_set_gamma);
+	const struct wlr_gamma_control_manager_v1_set_gamma_event *event = data;
+
+	struct sway_output *output = event->output->data;
+	output->gamma_lut_changed = true;
```
Comment (kennylevinsen) [inline sway/desktop/output.c:1100]: Was this state not meant to be cleared at some point?

Comment (Nefsen402) [reply sway/desktop/output.c:1100]: Indeed so. https://github.com/swaywm/sway/pull/7634

**Pull request merged**